src/app/app.module.ts

```diff
@@ -9,6 +9,7 @@ import { FooterComponent } from './components/footer/footer.component';
 import { BodyComponent } from './components/body/body.component';
 import { HomeComponent } from './components/home/home.component';
 import { ContactModalComponent } from './components/contact-modal/contact-modal.component';
+import { HttpClientModule } from '@angular/common/http';
 
 @NgModule({
```

Comment:
There is a merge conflict for this file, please resolve it

Reply:
i'll but for now i cannot run develop branch - some error with missing files of font awesome @AsafKaravani

Reply:
I have fixed it yesterday (reverted the last commit with all fontawesome files). Can you retry?

Reply:
trying it now, its not working - there's no  files of font awesome, maybe you forget to add them to git?

Reply:
OK, I get it. Let me try to resolve it.